integration/testnet/network.go

```diff
@@ -721,8 +721,8 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		RewardCut:                    cadence.UFix64(0),
 		CurrentEpochCounter:          cadence.UInt64(epochCounter),
 		NumViewsInEpoch:              cadence.UInt64(epochSetup.FinalView - epochSetup.FirstView - 1),
```

Comment:
```suggestion
		NumViewsInEpoch:              cadence.UInt64(epochSetup.FinalView - epochSetup.FirstView + 1),
```
The range is inclusive on both ends so we should add 1.